value/vector.go

```diff
@@ -15,13 +15,33 @@ import (
 	"robpike.io/ivy/config"
 )
 
-type Vector []Value
+type Vector struct {
+	slice []Value
```

Comment:
this should be called data, as it is in Matrix.
slice is a type, not a variable name.

Reply:
Done.